MSON Specification.md

```diff
@@ -436,6 +436,7 @@ A _Wildcard Type Name_ MAY only be used in a _[Type Specification][]_ for _[Memb
 #### 3.5.3 Type Attribute
 Defines extra attributes associated with the implementation of a type.
 
+- `nullable` - instance of this type *Value* MAY be unset (e.g. `null` or `nil`)
```

Comment:
Maybe we need to expand a bit more on `null` and `nil`? Maybe say that it is language specific? I will leave this to @zdne